Marlin/Configuration.h

```diff
@@ -1225,11 +1225,8 @@
 //
 //#define ULTRA_LCD   // Character based
 //#define DOGLCD      // Full graphics display
-
 #define ANET_KEYPAD_LCD
 //#define ANET_FULL_GRAPHICS_LCD
-// RepRap Discount (with Anet Adapter wiring see: http://www.thingiverse.com/thing:2103748)
-//#define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
```

Comment:
Thats true but readding it there was intentional. This is part of the SkyNet packaging changes. 

This also shows the problem in getting a Marlin PR done as that would imply that you can use the other LCDs with other boards too which is not the case.

I don't consider any changes to Configuration.h as part of a Marlin PR. There would be a "example configuration" for the ANet and this would probably contain these lines and none of all the other displays. But this is still to be decided. I would not change anything in the original Marlin Configuration.h. The one here is considered to be the "example anet" one.